doajtest/unit/resources/harvester_resp.json

```diff
@@ -0,0 +1 @@
": {"title": "My Journal", "medlineAbbreviation": "My Jour", "essn": "1234-5678", "issn": "9876-5432", "isoabbreviation": "My Jour", "nlmid": "123456789"}}, "pubYear": "2010", "pageInfo": "1", "affiliation": "aga@cottagelabs.com", "publicationStatus": "publish", "language": "eng", "pubModel": "Print-Electronic", "pubTypeList": {"pubType": ["Comment", "research-article", "Journal Article"]}, "fullTextUrlList": {"fullTextUrl": [{"availability": "Free", "availabilityCode": "F", "documentStyle": "html", "site": "PubMedCentral", "url": "https://www.harvester.test.articles/01"}]}, "commentCorrectionList": {"commentCorrection": [{"id": "12345678", "source": "MED", "reference": "My Jour. 2010 Jan;1(1):e1001011", "type": "Comment on", "orderIn": 1}]}, "isOpenAccess": "Y", "inEPMC": "Y", "inPMC": "Y", "hasPDF": "Y", "hasBook": "N", "hasSuppl": "N", "citedByCount": 0, "hasData": "N", "hasReferences": "N", "hasTextMinedTerms": "Y", "hasDbCrossReferences": "N", "hasLabsLinks": "N", "license": "cc by", "authMan": "N", "epmcAuthMan": "N", "nihAuthMan": "N", "hasTMAccessionNumbers": "N", "dateOfCompletion": "2011-08-19", "dateOfCreation": "2011-05-10", "firstIndexDate": "2011-05-11", "fullTextReceivedDate": "2020-07-09", "electronicPublicationDate": "2011-04-12", "firstPublicationDate": "2011-04-12"}, {"id": "12345678", "source": "MED", "pmid": "12345678", "pmcid": "PMC1234567", "fullTextIdList": {"fullTextId": ["PMC1234567"]}, "title": "Harvester Test Article  - doi and fulltext url missing", "authorString": "Domanska A.", "authorList": {"author": [{"fullName": "Domanska A.", "firstName": "Agnieszka", "lastName": "Domanska", "initials": "AMD"}]}, "journalInfo": {"issue": "1", "volume": "1", "journalIssueId": 111111, "dateOfPublication": "2010 Apr", "monthOfPublication": 1, "yearOfPublication": 2010, "printPublicationDate": "2010-01-01", "journal": {"title": "My Journal", "medlineAbbreviation": "My Jour", "essn": "1234-5678", "issn": "9876-5432", "isoabbreviation": "My Jour", "nlmid": "123456789"}}, "pubYear": "2010", "pageInfo": "1", "affiliation": "aga@cottagelabs.com", "publicationStatus": "publish", "language": "eng", "pubModel": "Print-Electronic", "pubTypeList": {"pubType": ["Comment", "research-article", "Journal Article"]}, "commentCorrectionList": {"commentCorrection": [{"id": "12345678", "source": "MED", "reference": "My Jour. 2010 Jan;1(1):e1001011", "type": "Comment on", "orderIn": 1}]}, "isOpenAccess": "Y", "inEPMC": "Y", "inPMC": "Y", "hasPDF": "Y", "hasBook": "N", "hasSuppl": "N", "citedByCount": 0, "hasData": "N", "hasReferences": "N", "hasTextMinedTerms": "Y", "hasDbCrossReferences": "N", "hasLabsLinks": "N", "license": "cc by", "authMan": "N", "epmcAuthMan": "N", "nihAuthMan": "N", "hasTMAccessionNumbers": "N", "dateOfCompletion": "2011-08-19", "dateOfCreation": "2011-05-10", "firstIndexDate": "2011-05-11", "fullTextReceivedDate": "2020-07-09", "electronicPublicationDate": "2011-04-12", "firstPublicationDate": "2011-04-12"}]}, "version": "6.5"}
```

Comment:
Very unimportant, but it might be best to pretty print this so we can easily inspect it by eye during development down the line